models/DataObject/Concrete/Dao/InheritanceHelper.php

```diff
@@ -233,10 +234,11 @@ public function doUpdate(int $oo_id, bool $createMissingChildrenRows = false, ar
                 $missingIds = $this->db->fetchFirstColumn($query);
 
                 // create entries for children that don't have an entry yet
-                $originalEntry = $this->db->fetchAssociative('SELECT * FROM ' . $this->querytable . ' WHERE ' . $this->idField . ' = ?', [$oo_id]);
+                $originalEntry = Helper::quoteDataIdentifiers($this->db, $this->db->fetchAssociative('SELECT * FROM ' . $this->querytable . ' WHERE ' . $this->idField . ' = ?', [$oo_id]));
```

Comment:
I don't think this is how `Helper::quoteDataIdentifiers()` should be used :thinking: